pkg/graph/schema.resolvers.go

```diff
@@ -44,6 +44,11 @@ func (r *existingModelLinkResolver) CurrentModelPlan(ctx context.Context, obj *m
 	return resolvers.ModelPlanGetByIDLOADER(ctx, *obj.CurrentModelPlanID) //TODO, implement loader, or this will be many queries
 }
 
+// Abbreviation is the resolver for the abbreviation field.
+func (r *modelPlanResolver) Abbreviation(ctx context.Context, obj *models.ModelPlan) (*string, error) {
+	panic(fmt.Errorf("not implemented: Abbreviation - abbreviation"))
```

Comment:
We have already synced about this, but just dropping a note so I get notified when you  have it updated! The field is named differently in model_plan.go, so this doesn't auto-resolve.